common/command.cc

```diff
@@ -120,6 +121,7 @@ po::options_description CommandHandler::getGeneralOptions()
 
 #endif
     general.add_options()("json", po::value<std::string>(), "JSON design file to ingest");
+    general.add_options()("write", po::value<std::string>(), "JSON design file to write");
```

Comment:
It would be a nice extra feature to be able to write JSON files after packing or placement as well as after routing - although perhaps this is best done as an option to disable routing altogether (for example) then just place, write JSON and exit.

Reply:
Added no-pack, no-place and no-route as options

Reply:
LGTM